test/benchmark/sqlite/ramfs_updates_individual/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+mkdir -p /tmp
+/benchmark/bin/sqlite-speedtest1 /tmp/test.db
```

Comment:
So it seems that this single benchmark program runs all the tests. This is very inefficient. And it results duplicated benchmark directories, in each of which the `run.sh` is the same. We should upgrade our benchmark framework so that it can collect multiple results in one run.

@grief8 Could you implement this much-needed feature to satisfy the usage of sqlite benchmark?

Reply:
Current github-action-benchmark can actually support submitting a benchmark containing multiple sets of results at one time. However, these data are treated as one benchmark, so our **alert_tool** will be either `customBiggerIsBetter` or `customBiggerIsBetter` and the benchmark chart will display all of the results. 

I think that since the current benchmark is run during idle time, repeated running will not bring particularly serious consequences.